### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by minsooshin**
landing_page

### CODE REVIEW

**Comment by minsooshin** [comment]
landing_page

File: src/components/UtilMenu/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        { isLogged && <MenuItem
+          primaryText='Sign Out'
+          onClick={onClickSign} /> }
+        { !isLogged && <MenuItem
+          primaryText='Sign In'
+          onClick={onClickSign} /> }
+        { !isLogged && <MenuItem
+          primaryText='Register'
+          onClick={() => this.routeToPage('register')} />
+        }
```
Comment (minsooshin) [inline src/components/UtilMenu/index.js]: I cannot do like this
```jsx
{ isLogged
? <div>
  <MenuItem />
  <MenuItem />
  <MenuItem />
</div>
: <div>
  <MenuItem />
  <MenuItem />
</div>
}
```
Due to the limitation of IconMenu of Material-ui, the MenuItem is wrapped with any other html element, it is not closed when the menu is clicked.

https://github.com/callemall/material-ui/issues/7119

**Comment by minsooshin** [comment]
landing_page_m

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import cx from 'classnames/bind'
+import style from './style.scss'
+
+const classNames = cx.bind(style)
+
+const Footer = () => (
+  <div className={classNames('Footer')}>
```
Comment (RaulZhao) [inline src/components/Footer/index.js]: If it is only static class names, how about use the string directly, like `className="Footer"`?

File: src/components/UtilMenu/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    const { isLogged, onClickSign, location } = this.props
+    const isRegister = location && location.pathname.includes('register')
+
+    return (
+      <div>
+        { !isRegister
+          ? <IconMenu
+            iconButtonElement={
+              <IconButton>
+                <Icon type='materialDesign' name='MdMoreVert' size={26} color="#fff" />
```
Comment (RaulZhao) [inline src/components/UtilMenu/index.js]: If we are suggested to use single quotes, then it should be `color='#fff'`

File: src/containers/Header/index.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      <AppBar
+        title={
+          <Link className={classNames('Header__brand')} to='/'>BaapChef</Link>
+        }
+        iconElementRight={
+          <UtilMenu
+            isLogged={this.state.logged}
+            onClickSign={this.onClickSign} />
+        }
+        style={{
```
Comment (RaulZhao) [inline src/containers/Header/index.js]: Can we use the library `material-ui/styles/colors` colors in .scss file? If yes, we should put the style setting to `style.scss`

File: src/containers/Register/index.js [hunk 5]
```diff
@@ -45,107 +47,111 @@ class Register extends Component {
...
+      <Row>
+        <Column small={12} medium={8} large={6} offset={{ medium: 2, large: 3}}>
+          <form onSubmit={this.handleFormSubmit}>
+            <TextField
+              type='text'
+              id='name'
+              name='name'
+              floatingLabelText='Restaurant Name'
+              fullWidth={true}
+              floatingLabelFocusStyle={{ color: lightBlue400 }}
```
Comment (RaulZhao) [inline src/containers/Register/index.js]: Can we add a classname to these `material-u` fields and then move the style settings to a `.scss` file?

File: src/containers/Header/index.js (hunk 4 above)
Comment (minsooshin) [reply src/containers/Header/index.js]: nope, we cannot do that. scss is totally separate file so it cannot be integrated with external library.

File: src/containers/Register/index.js (hunk 5 above)
Comment (minsooshin) [reply src/containers/Register/index.js]: No, we cannot. we can add className prop to the material-ui component, but how material-ui set the style is using inline style. So, we cannot set css property with class if the property is set by inline style from material-ui component.

**Pull request merged**